src/frequency.rs

```diff
@@ -113,9 +119,21 @@ impl<T: Eq + Hash> Extend<T> for Frequencies<T> {
     }
 }
 
+pub struct UniqueValues<'a, K> {
```

Comment:
This needs a doc string and should also be re-exported in `src/lib.rs` as part of the public API.

Reply:
Done